src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,27 @@
+package duke.task;
+
+/**
+ * A task of type deadline
```

Comment:
Perhaps need a full stop at the end? 🤔
And I noticed the same issue in several other places below.